atlassian/bitbucket/__init__.py

```diff
@@ -252,7 +252,10 @@ def upload_plugin(self, plugin_path):
     ################################################################################################
 
     def _url_projects(self, api_root=None, api_version=None):
+        if self.cloud:
+            return self.resource_url("repositories", api_root, api_version)
```

Comment:
This doesn*t make sense for me. Why is the URL for `projects` redirected `repositories`???

Reply:
Because this how api looks like in cloud. /projects route not supported in cloud.
https://developer.atlassian.com/bitbucket/api/2/reference/search?q=projects
https://developer.atlassian.com/bitbucket/api/2/reference/search?q=refs

Reply:
Thats because here it is called `workspace`. There is an open issue #604. If this API is needed I suggest to add it into the `Cloud` part of BitBucket.

Reply:
I agreed with you, from my point of view it should be 2 separate clients. But I currently didn`t have enough time for move it to cloud class or build new client. This is temporary solution, which allows persons like me use python client with bitbucket.org. Because it`s fully unusable in case when you want to do something with commits/tags/branches in cloud.